packages/zwave-js/src/lib/commandclass/SupervisionCC.ts

```diff
@@ -128,6 +128,7 @@ type SupervisionCCReportOptions = CCCommandOptions & {
 					| SupervisionStatus.NoSupport
 					| SupervisionStatus.Fail
 					| SupervisionStatus.Success;
+				duration: Duration;
 		  }
 	);
```

Comment:
The `duration` must be 0s for these `status` values according to the spec. Adding a duration field needlessly put the onus on the caller to follow the spec.

Reply:
Fixed :)